apps/barista/src/layout/smalltile/smalltile.ts

```diff
@@ -23,9 +23,36 @@ import { BaIndexPageItem } from '../../shared/page-contents';
   templateUrl: 'smalltile.html',
   styleUrls: ['smalltile.scss'],
   host: {
+    '[class]': '"ba-theme-" + _getTheme()',
     '[class.ba-smalltile-link-wrapper]': 'data',
   },
 })
 export class BaSmallTile {
   @Input() data: BaIndexPageItem;
+
+  /** @internal Gets the tile's theme based on its category. */
+  _getTheme(): string {
```

Comment:
Is there any specific reasoning behind, why this one and the `_identifier` are not getters?